assets/dev/js/editor/controls/media.js

```diff
@@ -44,9 +44,11 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		this.ui.controlMedia.toggleClass( 'elementor-media-empty', ! url );
 	},
 
-	openFrame: function() {
-		if ( ! FilesUploadHandler.isUploadEnabled( this.getMediaType() ) ) {
-			FilesUploadHandler.getUnfilteredFilesNotEnabledDialog( () => this.openFrame() ).show();
+	openFrame: function( e ) {
+		const mediaType = e?.target?.dataset?.value || this.getMediaType();
```

Comment:
It's better to call the parameter something like `media-type` rather than `value`